packages/web/migrations.json

```diff
@@ -1,4 +1,5 @@
 {
+  "cli": "nx",
```

Comment:
This is inconsistent but... this needs to move down to inside `update-babel-config-11.5.0`